Code/ThirdParty/zstd/cgmanifest.json

```diff
@@ -10,5 +10,6 @@
         }
       }
     }
-  ]
-}
\ No newline at end of file
+  ],
+  "version" : 1

```

Comment:
what does this do ?

Reply:
The azure open source checking tool now requires specifing which version the cgmanifest.json is.